tests/e2e/scale/test_scale_osds_fill_75%_reboot_workers.py

```diff
@@ -0,0 +1,103 @@
+"""
+This testcase starts with minimum number of osds(one osd)
+on each node and slowly scaling it into 6 osds and then reboot worker
+nodes
+"""
+
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.cluster import count_cluster_osd, validate_osd_utilization
+from ocs_ci.ocs.platform_nodes import AWSNodes
+from ocs_ci.ocs.node import get_typed_nodes, wait_for_nodes_status
+from ocs_ci.framework.testlib import scale, E2ETest, ignore_leftovers, tier4
+from ocs_ci.ocs.resources import storage_cluster
+from tests import helpers
+from ocs_ci.utility.utils import ceph_health_check
+
+
+logger = logging.getLogger(__name__)
+
+
+@scale
+@tier4
+@ignore_leftovers
```

Comment:
why ignore leftovers?

Reply:
We are scaling osds here, As we are not supporting cluster reduction for now, Added ignore_leftovers to ignore newly added OSDs.